### DISCUSSION TITLE
Sketchfab Models not showing my my Cloud instance Scene Editor

### DISCUSSION THREAD

**Discussion by Itspossible100pct** [mozilla/hubs#3311] [Hubs Cloud Questions] [answered]
Hello.  I have newly completed setup for my personal cloud server and inside Spoke/Scene Editor NONE of the Sketchfab or Google Poly assets are showing up in the asset browser.  Can anyone please advise on how to properly populate the assets browser in the cloud server?   
To be clear, everything shows up fine when I use Spoke in hubs.mozilla.com 

**_My Cloud instance Scene Editor_**
![My Cloud Scene Editor](https://user-images.githubusercontent.com/74311119/98828475-6b2fdf00-2438-11eb-9449-6d46cf94a19e.png)

**_Mozilla hubs spoke browser_**
![Moz Hubs - Spoke Browser](https://user-images.githubusercontent.com/74311119/98828463-679c5800-2438-11eb-8cc8-44409225b9e0.png)

**Comment by github-user-77** [comment, marked as answer]
@Itspossible100pct You need to provide API keys for Skecthfab and Google Poly for your Hubs Cloud instance. In your HC Admin panel, go to Server Settings – API Keys and put in the keys that you get from Sketchfab and Google Poly once you sign up with them.
